tests/compiled_templates/etcdv3/mesh/static-routes/bird_aggr.cfg

```diff
@@ -8,8 +8,8 @@ protocol static {
    route 192.168.221.192/26 blackhole;
    route 192.168.221.64/26 blackhole;
    # Static routes.
-   route 10.65.1.0/24 blackhole;
-   route 10.66.1.0/24 blackhole;
+   route 10.101.0.0/16 blackhole;
```

Comment:
Changed the value here so that it aligned with the cluster IP cidr that is used in the FV framework.